modules/model/actor/standard.js

```diff
@@ -105,18 +110,18 @@ export class StandardActorModel extends BaseActorModel {
         this.computeItems();
         super.computeDerived(items, flags);
         this.runScripts("computeCharacteristics", this.parent);
+        this.computeAdvantage();
+        this.computeMove();
+        this.computeSize();
```

Comment:
Why have these been moved?

Reply:
most important is to move ComputeSize before checkWounds, otherwise woundds calculation doesn't take into account Size.